src/include/block.h

```diff
@@ -233,6 +234,7 @@ struct __wt_bm {
     size_t handle_array_allocated; /* Size of handle array */
     WT_RWLOCK handle_array_lock;   /* Lock for block handle array */
     u_int handle_array_next;       /* Next open slot */
+    uint32_t objectid_complete;    /* Local objects at or below this id should be closed */
```

Comment:
I'm not crazy about this name. It relies on understanding the specific use of an object being "complete" defined elsewhere in the code.

I'm not sure I have a better suggestion. Maybe `objectid_max_flushed`? or `max_flushed_objectid`?

Reply:
Thanks, I went with `max_flushed_objectid`.